frontend/src/citizen-frontend/messages/MessagesPage.tsx

```diff
@@ -141,6 +145,17 @@ export default React.memo(function MessagesPage() {
                     onSend={(body) => sendMessage({ body })}
                     onSuccess={() => {
                       changeEditorVisibility(false)
+                      addAtomicNotification({
+                        children: (
+                          <div>
+                            {
+                              i18n.messages.messageEditor
+                                .messageSentNotification
+                            }
+                          </div>
+                        ),
+                        dataQa: 'message-sent-notification'
+                      })
```

Comment:
Voisi testata toimiiko myös pelkkä `children: i18n.messages.messageEditor.messageSentNotification` ilman `<div>`-hässäköitä.